src/components/ui/ProgressBar.js

```diff
@@ -0,0 +1,35 @@
+import { React } from 'react'
+import styled from 'styled-components/macro'
+
+import ProgressBarStep from '../ui/ProgressBarStep'
+
+const StyledProgressBar = styled.div`
+  display: flex;
+  justify-content: space-between;
+`
+const getStepStatus = (index, currentStep) => {
+  const refactoredIndex = index
+  if (refactoredIndex === currentStep) {
+    return 'active'
+  } else if (refactoredIndex < currentStep) {
+    return 'complete'
+  } else {
+    return 'incomplete'
+  }
+}
+
+const ProgressBar = ({ labels, className, currentStep, onChange }) => (
```

Comment:
Can you replace all mentions of currentStep with step? Don't think current is necessary anymore.